src/app/connections/list/list.component.html

```diff
@@ -1,47 +1,8 @@
 <ipaas-loading [loading]="loading">
-  <div class="container-cards-pf">
-    <div class="row row-cards-pf">
-      <div class="col-xs-12 col-sm-3 col-md-2 card" *ngFor="let connection of connections">
-        <div class="card-pf card-pf-view card-pf-view-xs">
-          <div class="card-pf-body">
-
-            <!-- Card Dropdown -->
-
-            <div class="card-pf-heading-kebab">
-              <div class="dropdown pull-right dropdown-kebab-pf card-pf-time-frame-filter">
-                <button type="button" class="btn btn-link dropdown-toggle" data-toggle="dropdown">
-                  <span class="fa fa-ellipsis-v"></span>
-                </button>
-                <ul class="dropdown-menu dropdown-menu-right" role="menu">
-                  <li>
-                    <a [routerLink]="[connection.id]">View</a>
-                  </li>
-                  <li>
-                    <a [routerLink]="[connection.id, 'edit']">Edit</a>
-                  </li>
-                  <li>
-                    <a>Duplicate</a>
-                  </li>
-                  <li>
-                    <a>Delete</a>
-                  </li>
-                </ul>
-              </div>
-
-              <!-- Card Title & Icon -->
-
-              <h2 class="card-pf-title">
-                <span class="fa {{ connection.icon }}"></span>
-                <a [routerLink]="[connection.id]" class="card-title">{{ connection.name }}</a>
-              </h2>
-            </div>
-
-            <!-- Card Description -->
-
-            <p>{{ connection.description | truncate : truncateLimit : truncateTrail }}</p>
-          </div>
-        </div>
-      </div>
-    </div>
+  <div class='connections list'>
+    <ipaas-connections-list-toolbar [childConnections]="connections | async"
+                                    (change)="updateConnections($event)"></ipaas-connections-list-toolbar>
+    <ipaas-connections-list-page [connectionList]="connections"
+                                 [loading]="loading"></ipaas-connections-list-page>
```

Comment:
Maybe this is because I haven't really looked at the integration editor designs, or maybe because I'm still thinking like Angular 1.x, but I'm confused about the previous approach. 1) Why would we be re-using lists if we are already going to be using a data store and a service to fetch the data? How many views will be using this same list to warrant it being its own component? Normally just a parent component for the Connections model, and child `list` and `detail` components suffice.* 2) The reason for this change is the idea that the parent is `list.component.ts` would be the only place that fetches the data and has several directives it passes that data to. In this case we can pass them whatever model we want. Looking at the way we organized files there was never any indication to me that the page was responsible for loading the data from the store and that it was the primary container. For that we can create a separate, reusable, agnostic, data-fetching, list-generating component that accepts any model and returns a list.

* The only exception to this are reusable components, like toolbars, that would be children components, and should definitely inherit from parent components that pass them whatever data you indicate it should have access to.